test/e2e/pkg/resources_test.go

```diff
@@ -175,13 +193,13 @@ var _ = Describe("Resources", Ordered, Label("e2e-tests-resources"), func() {
 			Consistently(func() error {
```

Comment:
Here is using Consistently. Consistently blocks when called for a specified duration. During that duration Consistently repeatedly polls its matcher and ensures that it is satisfied. If the matcher is consistently satisfied, then Consistently will pass. Otherwise Consistently will fail.

Reply:
yes, I updated the `Consistently` check to avoid intermittent error to break the check when getting nginx deployment.